### DISCUSSION TITLE
Filebrowser: paths yaml x settings.json

### DISCUSSION THREAD

**Discussion by Briare0s** [htpcBeginner/docker-traefik#270] [Q&A]
Hi there, thanks again for the amazing work. I'm learning so much while looking at your yaml file. I'm currently working with Filebrowser, but don't get my files to show up. I think it has to do with the way I defined my paths in the settings .json. 

Could you explain how you got your setup as depicted below working?

```
   volumes:
      - $DOCKERDIR/appdata/filebrowser/settings.json:/config/settings.json
      - $DOCKERDIR/appdata/filebrowser/filebrowser.db:/config/filebrowser.db
      - /media:/data/media
      - $USERDIR:/data/home
```

The [default settings](https://github.com/filebrowser/filebrowser/blob/master/docker/root/defaults/settings.json) as defined by Filebrowser themselves:

```
{
  "port": 80,
  "baseURL": "",
  "address": "",
  "log": "stdout",
  "database": "/database/filebrowser.db",
  "root": "/srv"
}
```

**Comment by Maynardnaze** [comment]
i think your missing this
      - $DATADIR:/srv

**Reply by Briare0s** [reply]
> i think your missing this - $DATADIR:/srv
> […](#)

This is htpcBeginner code in my first post, so the :/srv has to be defined differently, but I'm not sure how.
